backend/src/routes/user.ts

```diff
@@ -5,60 +5,37 @@ const router = new Router({
 	prefix: '/users',
 });
 
-//Get all users
 router.get('/', async (ctx, next) => {
 	ctx.body = await User.find();
 });
 
-//Find user by handle
 router.get('/:handle', async (ctx, next) => {
 	const user = await User.findOneBy({
-		handle: ctx.params.handle,
+		username: ctx.params.username,
 	});
 	if (user === null) {
 		ctx.status = 404;
 		return;
 	}
-
 	ctx.body = user;
 });
 
-//Create a new user with required handle and display name
-router.put('/newuser/:handle/:displayname', async (ctx, next) => {
-	console.log('new user route');
+router.put('/newuser', async (ctx, next) => {
```

Comment:
ok i think i misled you with my first review comment oops im sorry! we'll need a route for user creation and a different (but similar one) for updating information.
the one in the codebase right now ([see this line](https://github.com/rice-apps/tunedin/blob/develop/backend/src/routes/user.ts#L24)) can be used for creation, and we can have another one called `route.put('/:username/edit...` that will do essentially the same thing, but instead of checking if the user already exists, it'll check if the user doesn't exist before returning null.
this second route will be used for edit profile so it should update all user fields like you've already done here (for now, just username + displayname)